INSTALL.md

```diff
@@ -92,6 +137,20 @@ $ sudo apt-get install replicated replicated-ui replicated-agent replicated-upda
 
 You can check for new app releases by going to the management interface dashboard `https://waffle.company.com:8800` and clicking on the 'Check Now' button. If an update is available you will be able to read the release notes and install the update.
 
+#### Updating an Airgap installation
+
+To update an Airgap installation, you will need to download a new package via a link that we provide you and upload it to the server location specified in the "Console Settings" in the management ui.
+
+![Airgap Settings](doc/screenshots/7.png)
+
+##### On Host Machine
+```bash
+cd /home/ubuntu/updates
+wget --trust-server-names -O <your_update>.airgap "<link_we_provide>"
+```
```

Comment:
Given that this is an airgap install, they probably cannot do a `wget` from the host machine. They will probably have to do the `wget` on another machine outside the network, the `scp` it onto the host machine.